specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Define_package_reference_version.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Rules.MS_Build.Define_package_reference_version;
+
+public class Reports
+{
+    [Test]
+    public void on_no_output_type()
+       => new DefinePackageReferenceVersion()
+       .ForProject("PackageReferenceWithoutVersion.cs")
+       .HasIssue(
+           new Issue("Proj0804", "Define version for 'Warpstone' PackageReference."));
```

Comment:
0805: I'd like to use 0804 for misuse of Version when CPM is enabled (working on it right now).